npm/pkg/dataplane/ipsets/ipsetmanager_test.go

```diff
@@ -19,13 +18,20 @@ const (
 	testPodIP    = "10.0.0.0"
 )
 
-var iMgrApplyOnNeedCfg = &IPSetManagerCfg{
-	IPSetMode:   ApplyOnNeed,
-	NetworkName: "azure",
-}
+var (
+	iMgrApplyOnNeedCfg = &IPSetManagerCfg{
+		IPSetMode:   ApplyOnNeed,
+		NetworkName: "azure",
+	}
+
+	iMgrApplyAlwaysCfg = &IPSetManagerCfg{
+		IPSetMode:   ApplyAllIPSets,
+		NetworkName: "azure",
+	}
+)
 
 func TestCreateIPSet(t *testing.T) {
-	iMgr := NewIPSetManager(iMgrApplyOnNeedCfg, common.NewMockIOShim([]testutils.TestCmd{}))
+	iMgr := NewIPSetManager(iMgrApplyOnNeedCfg, getMockIOShim([]testutils.TestCmd{}))
```

Comment:
This won't work in Linux. Can we replace `getMockIOShim([]testutils.TestCmd{})` with 
```
calls := GetApplyIPSetsTestCalls(...) // already in testutils.go
common.NewMockIOShim(calls)
```
We would need to have as many `GetApplyTestCalls` as there are `iMgr.ApplyIPSets()` calls.

Since we will have to list out the toAddSets and toDeleteSets for `GetApplyIPSetsTestCalls`, can you run `assertEqualContentsTestHelper` for all the UTs in this file too?